src/index.ts

```diff
@@ -92,9 +92,21 @@ export function apply(ctx: Context, config: Config) {
 
       try {
         // send a message and wait for the response
-        const { conversationId, messageId } = conversations.get(key) ?? {}
-        const response = await api.sendMessage({ message: input, conversationId, messageId })
-        conversations.set(key, { conversationId: response.conversationId, messageId: response.messageId })
+        const conversation: Conversation = conversations.get(key) ?? { message: '', conversationId: undefined, messageId: undefined }
+        let action = 'next'
+        switch (input) {
+          case '重新说':
+              action = 'variant'
+            break
+          case '继续说':
+            action = 'continue'
+            break
+          default:
+            conversation.message = input
+        }
+        
```

Comment:
Create a lookup table for it, reduce the lines and human easily readable.

Reply:
What should I do here? I really don't have any ideas.🤣

Reply:
Something like this :

```ts
const actions = {
  重新说: 'variant',
  继续说: 'continue',
}

const action = input in actions ? actions[input] : 'next'
```

See? much clear and shorter than yours.

(maybe typescript would shout to you with some type errors, good luck!)